utils/swift_build_sdk_interfaces.py

```diff
@@ -105,7 +105,7 @@ def run_command(args, dry_run):
     try:
         out, err = proc.communicate()
```

Comment:
Does setting `os.environ['PYTHONIOENCODING'] = 'UTF-8'` obviate the code below?

Reply:
I didn't know about `PYTHONIOENCODING`, so thanks for telling me about that.  Unfortunately, in my testing, it seems that it does not obviate the change :-(.  Would you like me to add that as well?

Reply:
Nah, this is fine on its own then.